lib/playback/skipnext.ts

```diff
@@ -0,0 +1,34 @@
+import putSpotifyData from "@/lib/putSpotifyData";
+
+import getAccessToken from "../getAccessToken";
+
+const next = async (
+  uri: string,
+  isTrackPlaying: boolean,
+  isSkipToPrevious = false
+) => {
+  let endpoint = "https://api.spotify.com/v1/me/player/play";
+  let method = "PUT";
+  let body: string | null = JSON.stringify({
+    uris: [uri],
+  });
+
+  if (isSkipToPrevious) {
```

Comment:
What does this do? Isn't this the goToNextSong function? Can you add JSDoc comments